### DISCUSSION TITLE
SVG Path.PathPoints aren't the same as they are in actual SVG file.

### DISCUSSION THREAD

**Discussion by IgnEriol** [vvvv/SVG#1171] [Q&A]
Hello! I'm not sure if this behavior is correct or not, maybe I do not have enough knowledge.
In my SVG source file I see point coordinates like that: 
-262.5 -4.23 -127.01 -12.71 -127.01 -21.16
![Style 4 15 ProMax](https://github.com/user-attachments/assets/10c3c278-838b-4d5d-be68-2758464c10c7)

But after reading it with SvgDocument.Open I'm getting different values (seems to me they're too big):
![image](https://github.com/user-attachments/assets/f99e9e49-3461-44c7-b919-5a86239c9f19)

Is it correct and should be working as expected or it is some bug?

SVG version - 3.4.7
net8.0-windows

**Comment by benberlin7** [comment]
The path coordinates can work as absolute (big letteR) or relative commands.
To get a better understanding use your svg path via https://svg-path-visualizer.netlify.app/
So in case the commands are relative it could lead to very different absolute coordinates.
